src/main.js

```diff
@@ -6,68 +6,96 @@ const components = jsonfile.readFileSync('components.json')
 
 const q = readlineSync.question
 
-let format = q(`Choose a format [${Object.keys(formats["Ro-sham-bo"])}] `)
-if (format === "") {
-  format = "standard"
-}
-// hardcode game and mode
-const game = "Ro-sham-bo"
-let mode = q("Choose a mode (default: leaderboard) ")
-if(mode === "") {
-  mode = "leaderboard"
-}
+// choose game
+let game = q("Choose a game [Ro-sham-bo, Double Ro-sham-bo]: ")
+if (game === "") { game = "Ro-sham-bo" }
+
+// choose format
+let format = q(`Choose a format [${Object.keys(formats)}]: `)
+if (format === "") { format = "standard" }
+
+// choose mode
+let mode = q("Choose a mode [leaderboard, jesseMillerOnly]: ")
+if (mode === "") { mode = "leaderboard" }
 
-const gameComponents = formats[game][format]
+const gameComponents = formats[format]
 
 const getRules = (answer) => {
   return {
     choice: answer,
-    rules: components[game][answer]
+    rules: components[answer]
   }
 }
 
 const findWinner = ([p1Name, p1], [p2Name, p2]) => {
-  let p1Wins = false
-  let p2Wins = false
 
   // Check all the options of which choice wins and loses against the other
   // choice
   if (p1.rules.winsAgainst.includes(p2.choice)) {
-    p1Wins = true
+    return p1Name
   }
   if (p2.rules.winsAgainst.includes(p1.choice)) {
-    p2Wins = true
+    return p2Name
   }
   if (p1.rules.losesAgainst.includes(p2.choice)) {
-    p2Wins = true
+    return p2Name
   }
   if (p2.rules.losesAgainst.includes(p1.choice)) {
-    p1Wins = true
+    return p1Name
   }
 
-  // if there is a unique winner return that winner
+  // if there is no unique winner:
+  return false
 
-  if (p1Wins === p2Wins) {
-    return false
+}
+
+const findDoubleWinner = ([p1Name, p1a, p1b], [p2Name, p2a, p2b]) => {
+  // play the players' 1st and 2nd choices against each other like regular
+  // Ro-sham-bo
+  round1 = findWinner([p1Name, p1a], [p2Name, p2a])
+  round2 = findWinner([p1Name, p1b], [p2Name, p2b])
+
+  // Add up the number of rounds each player won
+  let p1Score = 0
+  let p2Score = 0
+  if (round1 === p1Name) {
+    ++p1Score
```

Comment:
This form of increment is generally considered dangerous and confusing (at least by the airbnb style guide). The recommend `p1Score += 1` for it's expressivity and lack of confusing side effects.